src/ligero/mod.rs

```diff
@@ -161,13 +156,10 @@ where
         let mat = Matrix::new_from_flat(m, m, &coeffs);
 
         // 2. Apply Reed-Solomon encoding row-wise
-        let fft_domain = GeneralEvaluationDomain::<F>::new(m).unwrap();
-        let mut domain_iter = fft_domain.elements();
-
         let ext_mat = Matrix::new_from_rows(
             mat.rows()
                 .iter()
-                .map(|r| reed_solomon(r, rho_inv, fft_domain, &mut domain_iter))
+                .map(|r| reed_solomon(r, rho_inv))
```

Comment:
I trust you removing these two parameters. Next time we can discuss why you no longer think passing them will lead to some efficiency gain.

Reply:
As I wrote in the PR description, I'm not sure whether this is optimal and that we might want to bring it back. Intuitively though, finding an FFT domain should be inexpensive compared to m*log(m) cost of the FFT itself for large enough m, since the constructor doesn't compute all the elements explicitly.
For now I'd leave it as-is, later we bench and see. I'll create an issue so we don't forget

Reply:
https://github.com/HungryCatsStudio/poly-commit/issues/13